lib/services.js

```diff
@@ -127,42 +118,12 @@ var mSvcConfigs = {
         sharded: false,
         imageNames: ['mantav2-loadbalancer', 'manta-loadbalancer']
     },
-    jobsupervisor: {
-        oneach: true,
-        probes: true,
-        sharded: false,
-        imageNames: ['mantav2-jobsupervisor', 'manta-jobsupervisor']
-    },
-    jobpuller: {
-        oneach: true,
-        probes: true,
-        sharded: false,
-        imageNames: ['mantav2-jobpuller', 'manta-jobpuller']
-    },
-    medusa: {
-        oneach: true,
-        probes: true,
-        sharded: false,
-        imageNames: ['mantav2-medusa', 'manta-medusa']
-    },
     ops: {
         oneach: true,
         probes: true,
```

Comment:
Could drop `probes: true` and `oneach: true` for all of these now right?

Reply:
probably.  I'll try.

Reply:
seems so!